src/utils/app/data/data-service.ts

```diff
@@ -285,7 +288,7 @@ export class DataService {
             id: constructPath(relativePath, folder.name),
             name: folder.name,
             type: FolderType.File,
-            absolutePath: constructPath('files', bucket, relativePath),
+            absolutePath: constructPath(ApiKeys.Files, bucket, relativePath),
```

Comment:
This is not same as in row 222 - here we constructing backend URL and it will be files/bucket/... but you are replacing it with files/file/bucket/...